template.php

```diff
@@ -112,7 +112,7 @@ function europa_menu_tree__secondary(&$variables) {
  * Implements hook_bootstrap_colorize_text_alter().
  */
 function europa_bootstrap_colorize_text_alter(&$texts) {
-  $texts['contains'][t('Save')] = 'primary';
+  $texts['contains'][t('Save', [], ['langcode' => $GLOBALS['language_content']->language])] = 'primary';
```

Comment:
@nelsongrin, from what i see the changes you made here are completely useless.
This hook basically defines a mapping between strings and classes in order to return a certain class if a match is found.